### DISCUSSION TITLE
KeyError: "None of [Index(['tic', 'date', 'macd'], dtype='object')] are in the [columns]"

### DISCUSSION THREAD

**Discussion by Boaz1248** [AI4Finance-Foundation/FinRL#1122] [General]
![image](https://github.com/AI4Finance-Foundation/FinRL/assets/137479347/48be0c27-6a9d-414a-9f5e-dc990fcd7169)

Hi, is there any way to overcome this issues? I faced this error with only FX Ticker. But it works with only one pair not multiple. 

Thanks

**Comment by mw66** [comment]
You may using pandas 2.0+, try: `pip install "pandas<2"`
